File: wastage-reduction-final-master/registration-service/src/main/java/com/stackroute/registrationserver/domain/DeliveryBoyProfile.java
package com.stackroute.registrationserver.domain;


import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.Entity;
import javax.persistence.Id;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class DeliveryBoyProfile {

    @Id
    private String username;
    private String email;
    private String role;
    private String name;
    private Long mobile;
    private String address;
    private String licenseNo;
    private String licenseName;
}

File: wastage-reduction-final-master/feedol-ui/src/app/deliveryBoy-profile.ts
export class DeliveryBoyProfile{
​
    username:string;
    email:string;
    role:string;
    name:string;
    mobile:string;
    address:string;
    licenseNo:string;
    licenseName:string
​
    constructor(username, email, role, name, mobile, address,  licenseNo,licenseName){
        this.username = username;
        this.email = email;
        this.role = role;
        this.name = name;
        this.mobile = mobile;
        this.address = address;
        this.licenseNo = licenseNo;
        this.licenseName = licenseName;
    }
}